stylus/ui-app/modals.styl

```diff
@@ -0,0 +1,63 @@
+/*!
+ * NOTIFICATIONS
+ * =============
+ *
+ * This file contains all needs for modals
+ */
+
+
+$modals
+    modal-width = em(480px)
+    .coz-modal
+        z-index 100
+
+    .coz-overlay
+        position   absolute
+        height     100%
+        width      100%
```

Comment:
Could you just put together declaration of the same thematic so it fits the guidelines? In this case, top & left before width & height. 
Thank you 😄